__tests__/app.test.js

```diff
@@ -142,6 +142,7 @@ describe('Endpoint Tests', () => {
                     expect(body).toHaveProperty("created_at")
                     expect(body).toHaveProperty("votes")
                     expect(body).toHaveProperty("article_img_url")
+                    expect(body).toHaveProperty("comment_count")
```

Comment:
We shouldn't be amending a previous test here and should be doing a new test to show that a single article object has the comment count included 👍